packages/styleguide/src/components/Link/Link.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import styled from 'styled-components';
+import cx from 'classnames';
+
+const CLASS_ROOT = 'link';
```

Comment:
Styleguide componenty naschval nemaju citatelnu classname ako napr. `link` aby to nekolidovalo s projektovym css. Teraz moze nastane kolizia a `<Link />` podedi styly z projektu ak tam bude tiez classa `.link`

Reply:
ok vymažem teda class names :) budem vedieť.